scripts/vfx/build_oiio_util.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) Contributors to the aswf-docker Project. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -ex
+
+git clone https://github.com/OpenImageIO/oiio.git
+cd oiio
+
+if [ "$OIIO_VERSION" != "latest" ]; then
+    git checkout tags/Release-${OIIO_VERSION} -b Release-${OIIO_VERSION}
+fi
+
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=${ASWF_INSTALL_PREFIX} \
+      -DOIIO_BUILD_TOOLS=ON \
```

Comment:
You could pass an environment variable to set this flag instead of creating a whole new script?